### ISSUE TITLE
[Bug]: Files outside of progression do not fetch.

### ISSUE THREAD

**Issue by Surasia** [Cubicpath/HaloInfiniteGetter#72]
### Contact Details

surasiaarchive@gmail.com

### What happened?

When trying to scan endpoints such as https://gamecms-hacs.svc.halowaypoint.com/hi/Progression/guide/xo?flight=[flightID], a 403 error is returned, as that directory does not exist (404s are for files in waypoint API). The cause of this is that InfiniteGetter tries to fetch https://gamecms-hacs.svc.halowaypoint.com/hi/progression/file/guide/xo?flight=0ca7dc01-0016-427f-b765-13bf56f3722e, adding an additional /file/ in front of the path.

### Version

v0.12.2

### What operating system are you using?

Windows

### Relevant log output

```shell
Could not get resource at:
https://gamecms-hacs.svc.halowaypoint.com/hi/progression/file/guide/xo?flight=0ca7dc01-0016-427f-b765-13bf56f3722e

Reason:
-----------------------------------
403: Forbidden
Request forbidden -- You cannot get this resource with or without an API token.
```

**Comment by Cubicpath** [comment]
I always assumed that all endpoints within `progression` had a `/file/` path appended. Is this not the case? The example above does not work for me, both https://gamecms-hacs.svc.halowaypoint.com/hi/progression/guide/xo?flight=0ca7dc01-0016-427f-b765-13bf56f3722e and https://gamecms-hacs.svc.halowaypoint.com/hi/guide/xo?flight=0ca7dc01-0016-427f-b765-13bf56f3722e return a `403`.

**Comment by Surasia** [comment]
?flight= needs a clearance token which can be fetched using the settings endpoint. There's also some other misc files which use hi/multiplayer for instance.

**Comment by Cubicpath** [comment]
Fixed in #78
